google/oauth2/_client.py

```diff
@@ -43,6 +43,10 @@
     "https://iamcredentials.googleapis.com/v1/"
     + "projects/-/serviceAccounts/{}:generateIdToken"
 )
+_TRUST_BOUNDARY_LOOKUP_ENDPOINT = (
```

Comment:
This seems to be duplicated in service account

Reply:
It is okay for oauth2 module to depend on auth module.

Reply:
My bad to not delete this after I put all the default in the `auth.credentials`